CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+## 3.10.0 (Unreleased)
+
+FEATURES:
+
+* resource/gitlab_project_hook: added `releases_events` attribute (#773)
+* resource/gitlab_user: added `state` attribute to allow blocking users (#762)
+* resource/gitlab_project: added `ci_forward_deployment_enabled` attribute (#732)
+* datasource/gitlab_project: added `ci_forward_deployment_enabled` attribute (#732)
+* datasource/gitlab_projects: added `ci_forward_deployment_enabled` attribute (#732)
+* resource/gitlab_group_access_token: added support for Group Access Tokens (#860)
+* datasource/gitlab_group_membership: support pagination (#858)
+
+BUG FIXES
+
+* resource/gitlab_project: correctly handle push rules add and edit (#838)
+* datasource/gitlab_projects: allow to get archived and unarchived repositories (#855)
+* resource/gitlab_group: support setting `default_branch_protection` to `0` (#856)
+* resource/gitlab_project: support creating project in group without default branch protection (#856)
+
+ENHANCEMENTS:
+
+* docs: better provider introduction (#852)
+
+NOTES:
+
+* provider: aligned project structure to terraform provider scaffolding project (#815)
+* repository: changed default branch from `master` to `main` (#852)
+* repository: introduction of multiple GitHub workflows to improve developer experience
+
```

Comment:
We only need to include items in the changelog which affect users of the provider. Otherwise it can create confusion. I think all of these can be cut. Other examples of things we can omit would be testing changes and refactors (usually).

```suggestion
```

Reply:
Okay, sounds good :)